tsl/test/isolation/expected/continuous_aggs_multi.out

```diff
@@ -94,7 +94,8 @@ step Setup2:
     END; $$ LANGUAGE plpgsql;
 
 step AlterLag1: alter view continuous_view_1 set (timescaledb.refresh_lag = 10);
-R1: LOG:  materializing continuous aggregate public.continuous_view_1: nothing to invalidate, new range up to 15
+ERROR:  cannot alter continuous aggregate using ALTER VIEW
```

Comment:
This error needs to be fixed. It alters the output for the .out file.

Reply:
Fixed.